### DISCUSSION TITLE
Can't reverify my academic status

### DISCUSSION THREAD

**Discussion by ArjixWasTaken** [community/community#51346] [GitHub Education]
My university doesn't provide a school-issued email address to undergraduate students.
That is not a problem since I can submit a document instead, but that no longer works, since I can't even move onto the next step of the process.

I am stuck on 

![image](https://user-images.githubusercontent.com/53124886/228599231-ce37376b-bb52-49dc-9251-1520fd607a62.png)

Back in January this worked just fine
![image](https://user-images.githubusercontent.com/53124886/228599435-498c01eb-361b-4907-9d40-5cd9efb96ed1.png)

Please help

**Comment by samus-aran** [comment]
Hi @ArjixWasTaken 👋 

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient. In this case it would be the Academic email(s) which has been associated with this school. 

We would recommend to you speak to your institution regarding to acquiring an email address to proceed.

**Reply by ArjixWasTaken** [reply]
https://www.unipi.gr/unipi/el/hu-proxh-mail.html

![image](https://user-images.githubusercontent.com/53124886/229094614-00d60bd5-b429-4a1a-8979-c483cfc56cd5.png)
![image](https://user-images.githubusercontent.com/53124886/229094741-38ac644b-9544-4847-a23a-999b9e744191.png)

**Reply by ArjixWasTaken** [reply]
I can attempt to ask for a school-issued mail, but as the University's site says, they don't give it to Undergraduate students.
My university uses my personal email for communicating with me.
